ui-ngx/src/app/shared/components/help-popup.component.html

```diff
@@ -32,17 +32,21 @@
 </fieldset>
 <fieldset class="tb-help-popup-button-container" *ngIf="textMode">
   <div #toggleHelpTextButton
+       [ngClass] = "{'hint-button': hintMode}"
        (click)="toggleHelp()">
     <button mat-button
             color="primary"
             class="tb-help-popup-text-button"
-            [ngClass]="{'mat-mdc-outlined-button mdc-button--outlined': popoverVisible && popoverReady}">
+            [ngClass]="{'mat-mdc-outlined-button mdc-button--outlined': popoverVisible && popoverReady,
+                        'tb-standard-help-popup-text-button': !hintMode,
+                        'tb-hint-help-popup-text-button': hintMode}">
```

Comment:
Perhaps it is better to leave the style classes as they were and override the style values with a new CSS class?